airbyte-cdk/python/unit_tests/sources/declarative/parsers/test_model_to_component_factory.py

```diff
@@ -732,7 +731,12 @@ def test_create_default_paginator():
     resolved_manifest = resolver.preprocess_manifest(parsed_manifest)
     paginator_manifest = transformer.propagate_types_and_options("", resolved_manifest["paginator"], {})
 
-    paginator = factory.create_component(model_type=DefaultPaginatorModel, component_definition=paginator_manifest, config=input_config)
+    paginator = factory.create_component(
```

Comment:
Can you add to the `test_full_config_stream()` test to replace `NoPagination` w/ a default paginator + offset/page increment pagination strategy. That way we can test that the `url_base` is properly passed down to the paginator by verifying the field is set

Reply:
The `list_stream` in this test already have a DefaultPaginator. I updated the config to remove the url_base